roles/openshift_excluder/README.md

```diff
@@ -1,50 +1,69 @@
 OpenShift Excluder
-================
+==================
 
 Manages the excluder packages which add yum and dnf exclusions ensuring that
-the packages we care about are not inadvertantly updated. See
+the packages we care about are not inadvertently updated. See
 https://github.com/openshift/origin/tree/master/contrib/excluder
 
 Requirements
 ------------
-openshift_facts
 
+None
 
-Facts
------
+Inventory Variables
+-------------------
 
-| Name                       | Default Value | Description                            |
------------------------------|---------------|----------------------------------------|
-| enable_docker_excluder     | enable_excluders | Enable docker excluder. If not set, the docker excluder is ignored. |
-| enable_openshift_excluder  | enable_excluders | Enable openshift excluder. If not set, the openshift excluder is ignored. |
-| enable_excluders           | None             | Enable all excluders
+| Name                                 | Default Value              | Description                            |
+---------------------------------------|----------------------------|----------------------------------------|
+| openshift_enable_excluders           | True                       | Enable all excluders                   |
+| openshift_enable_docker_excluder     | openshift_enable_excluders | Enable docker excluder. If not set, the docker excluder is ignored. |
+| openshift_enable_openshift_excluder  | openshift_enable_excluders | Enable openshift excluder. If not set, the openshift excluder is ignored. |
 
 Role Variables
 --------------
-None
```

Comment:
I stopped short of requiring every variable to be defined when calling the role.  It was a trade-off between having everything being passed in and usability.

If we required everything to be passed in, it would look something like this, with three extra `_enable_` vars passed each time the role is called even though they are not necessarily providing value to what is being done.  We either put the inventory variables only in role defaults, or we put them in every role call.

```yaml
- name: Demonstrate OpenShift Excluder usage
  hosts: oo_masters_to_config:oo_nodes_to_config
  roles:
  # Disable all excluders
  - role: openshift_excluder
    r_openshift_excluder_action: disable
    r_openshift_excluder_enable_excluders: "{{ openshift_enable_excluders | default(enable_excluders) | default(true) }}"
    r_openshift_excluder_enable_openshift_excluder: "{{ openshift_enable_openshift_excluder | default(enable_openshift_excluder) | default(r_openshift_excluder_enable_excluders) }}"
    r_openshift_excluder_enable_docker_excluder: "{{ openshift_enable_docker_excluder | default(enable_docker_excluder) | default(r_openshift_excluder_enable_excluders) }}"
    r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
  # Enable all excluders
  - role: openshift_excluder
    r_openshift_excluder_action: enable
    r_openshift_excluder_enable_excluders: "{{ openshift_enable_excluders | default(enable_excluders) | default(true) }}"
    r_openshift_excluder_enable_openshift_excluder: "{{ openshift_enable_openshift_excluder | default(enable_openshift_excluder) | default(r_openshift_excluder_enable_excluders) }}"
    r_openshift_excluder_enable_docker_excluder: "{{ openshift_enable_docker_excluder | default(enable_docker_excluder) | default(r_openshift_excluder_enable_excluders) }}"
    r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
  # Disable all excluders and verify appropriate excluder packages are available for upgrade
  - role: openshift_excluder
    r_openshift_excluder_action: disable
    r_openshift_excluder_enable_excluders: "{{ openshift_enable_excluders | default(enable_excluders) | default(true) }}"
    r_openshift_excluder_enable_openshift_excluder: "{{ openshift_enable_openshift_excluder | default(enable_openshift_excluder) | default(r_openshift_excluder_enable_excluders) }}"
    r_openshift_excluder_enable_docker_excluder: "{{ openshift_enable_docker_excluder | default(enable_docker_excluder) | default(r_openshift_excluder_enable_excluders) }}"
    r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
    r_openshift_excluder_verify_upgrade: true
    r_openshift_excluder_upgrade_target: "{{ openshift_upgrade_target }}"
    r_openshift_excluder_package_state: latest
    r_openshift_excluder_docker_package_state: latest
```